scripts/pysa_fuzzer/example_generation.py

```diff
@@ -0,0 +1,104 @@
+import random
+import math
+a = input()

```

Comment:
What you generate should roughly match the format of our integation tests here: https://github.com/facebook/pyre-check/tree/main/source/interprocedural_analyses/taint/test/integration

Ideally it is a file with some function calls and 1 valid flow per file